.github/workflows/gh-pages.yml

```
@@ -0,0 +1,22 @@
+name: GitHub Pages
+run-name: ${{ github.actor }} is rolling out a new version of the app 🚀
+on: [push]
+jobs:
+  Build-and-Deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 16
+    - name: Install npm deps
+      working-directory: ./react-chat
+      run: npm i
+    - name: Build app
+      working-directory: ./react-chat
+      run: npm run build
+    - name: Deploy
+      uses: peaceiris/actions-gh-pages@v3
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        publish_dir: ./public
```

Нужно сделать так, потому что в исходниках ничего полезного нет
```suggestion
        publish_dir: ./react-chat/build
```